modules/core/include/opencv2/core/mat.hpp

```diff
@@ -987,8 +987,7 @@ class CV_EXPORTS Mat
 #ifdef CV_CXX11
     /** @overload
     */
-    template<typename _Tp, typename = typename std::enable_if<std::is_arithmetic<_Tp>::value>::type>
-    explicit Mat(const std::initializer_list<_Tp> list);
+    template<typename _Tp> explicit Mat(const std::initializer_list<_Tp> list);
```

Comment:
Nope.
Check this PR: #9507

Reply:
Thank you for teaching me a lot.
I didn't understand why this constructor was added, but I could understand it now.   I think I will close this pull request, what do you think?
>All merge commits should be eliminated from the patch.
-> I understood.

Reply:
Prior to sending the pull request, I should have confirmed whether it was a problem and should receive advice.